src/copy.c

```diff
@@ -447,7 +447,17 @@ TSCmpBuffersByUsage(const ListCell *a, const ListCell *b)
 	Assert(b1 >= 0);
 	Assert(b2 >= 0);
 
-	return (b1 > b2) ? 1 : (b1 == b2) ? 0 : -1;
+	if (b1 > b2)
+	{
+		return 1;
+	}
+
+	if (b1 == b2)
+	{
+		return 0;
+	}
+
+	return -1;
```

Comment:
```suggestion
  return pg_cmp_s32(b1, b2);
```

Reply:
This function was introduced in PG17, we can't use it.